mkdocs.yml

```diff
@@ -42,7 +42,13 @@ nav:
                   - Hugging Face: modelserving/v1beta1/triton/huggingface/README.md
                 - AMD: modelserving/v1beta1/amd/README.md
               - LLM Runtime:
-                - Hugging Face LLM: modelserving/v1beta1/llm/huggingface/README.md
+                - Hugging Face LLM:
+                  - Overview: modelserving/v1beta1/llm/huggingface/README.md
+                  - Text Generation: modelserving/v1beta1/llm/huggingface/text_generation/README.md
+                  - Text2Text Generation: modelserving/v1beta1/llm/huggingface/text2text_generation/README.md
+                  - Token Classification: modelserving/v1beta1/llm/huggingface/token_classification/README.md
+                  - Text Classification: modelserving/v1beta1/llm/huggingface/text_classification/README.md
+                  - Fill Mask: modelserving/v1beta1/llm/huggingface/fill_mask/README.md
```

Comment:
Shall we separate out the non generation tasks? Two options:
- Add everything under Hugging Face runtime
- Separate out the docs for Hugging Face vLLM backend and HF backend (token/text classification)